doc/developer-guide/api/functions/TSMgmtSourceGet.en.rst

```diff
@@ -0,0 +1,69 @@
+.. Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed
+   with this work for additional information regarding copyright
+   ownership.  The ASF licenses this file to you under the Apache
+   License, Version 2.0 (the "License"); you may not use this file
+   except in compliance with the License.  You may obtain a copy of
+   the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+   implied.  See the License for the specific language governing
+   permissions and limitations under the License.
+
+.. include:: ../../../common.defs
+
+.. default-domain:: c
+
+TSMgmtSourceGet
+***************
+
+Synopsis
+========
+
+`#include <ts/ts.h>`
+
+.. function:: TSReturnCode TSMgmtSourceGet(const char * var_name, TSMgmtSource * result)
+
+Description
+===========
+
+Get the source of a value for a configuration variable. :arg:`var_name` is the name of the variable
+as a nul terminated string. The source value is stored in :arg:`result`. The function can return
+failure if :arg:`var_name` is not found.
+
+Types
+=====
+
+.. type:: TSMgmtSource
+
+   Source of the current value for a management (configuration) value.
+
+   .. macro:: TS_MGMT_SOURCE_NULL
```

Comment:
Consistency with all of the other `TSMgmt...Get()` methods. See below. I actually had the function that way originally and changed it back for this reason.
```
tsapi TSReturnCode TSMgmtIntGet(const char *var_name, TSMgmtInt *result);
tsapi TSReturnCode TSMgmtCounterGet(const char *var_name, TSMgmtCounter *result);
tsapi TSReturnCode TSMgmtFloatGet(const char *var_name, TSMgmtFloat *result);
tsapi TSReturnCode TSMgmtStringGet(const char *var_name, TSMgmtString *result);
tsapi TSReturnCode TSMgmtSourceGet(const char *var_name, TSMgmtSource *result);
// vs.
tsapi TSMtmtSource TSMgmtSourceGet(const char *var_name);
```